### ISSUE TITLE
Babel pipeline config did not work as instructed in tutorial "Design Systems for Developers"

### ISSUE THREAD

**Issue by TamaraPK** [chromaui/learnstorybook.com#440]
Location: [here](https://www.learnstorybook.com/design-systems-for-developers/react/en/distribute/)

I had to run the following:

`yarn add --dev @babel/cli cross-env

In project root add file ".babelrc" with the content below:

`{ "presets": [[ "react-app", { "absoluteRuntime": false } ]] }`

 

Otherwise the given tutorial instructions on the page gave an error:
absoluteRuntime option not found

**Comment by TamaraPK** [comment]
Update: I have corrected the above as my previous settings resulted in absolute oath errors. The instructions in the tutorial still did not work but adding the .babelrc file as shown above works :)

**Comment by jonniebigodes** [comment]
@TamaraPK going to close this now as the tutorial was updated to reflect the issues you've encountered. 

Regarding this particular issue, I wasn't able to replicate it. I ran the tutorial and code from end to end and that particular issue didn't pop in. This could be related to a node version difference or some other configuration you have setup on your end. Let us know if the issue still persists.
